cni/pkg/iptables/iptables.go

```diff
@@ -405,13 +406,19 @@ func (cfg *IptablesConfigurator) executeCommands(iptablesBuilder *builder.Iptabl
 
 func (cfg *IptablesConfigurator) executeIptablesCommands(iptVer *dep.IptablesVersion, args [][]string) error {
 	var iptErrs []error
+	// TODO: pass log all the way through
```

Comment:
Why not pass all the way through, if we are going to the trouble of adding a scope just for iptables and passing it this far?

Reply:
Its used in a pretty large number of places and in interfaces. TBH we should I was just lazy

Reply:
Fair enough,  that's why I didn't scope it to begin with.